packages/sessions-sdk-react/src/field.tsx

```diff
@@ -0,0 +1,88 @@
+import clsx from "clsx";
+import type { ReactNode, ComponentProps } from "react";
+import {
+  FieldError,
+  Input,
+  Label,
+  TextField as TextFieldImpl,
+  NumberField as NumberFieldImpl,
+  Text,
+  Group,
+  Button,
+} from "react-aria-components";
+
+import styles from "./field.module.css";
+
+export const TextField = ({
+  label,
+  description,
+  className,
+  controls,
+  ...props
+}: ComponentProps<typeof TextFieldImpl> & {
+  label?: ReactNode | undefined;
+  description?: ReactNode | undefined;
+  controls?: ReactNode;
+}) => (
+  <TextFieldImpl className={clsx(styles.textField, className)} {...props}>
+    <Field label={label} description={description}>
+      <Input className={styles.input ?? ""} />
+      {controls && <div className={styles.controls}>{controls}</div>}
+    </Field>
+  </TextFieldImpl>
+);
+
+export const NumberField = ({
```

Comment:
seems unused

Reply:
I originally planned to use this but then ended up doing something else, however I decided to keep the code because it will be tree-shaken out and it will likely be a useful component down the line